src/Jackett.Common/Definitions/divteam.yml

```diff
@@ -120,15 +120,15 @@ search:
       optional: true
       filters:
         - name: append
-          args: " English"
+          args: " English-DivTeam"
     title:
       selector: a[href^="index.php?page=torrent-details"][onmouseover]:not(:contains("VOSE"))
       optional: true
       filters:
         - name: re_replace
           args: ["\\W", "."] # Spaces and other characters -> .
         - name: append
-          args: ".Spanish-DivTeam"
+          args: ".MULTi/Spanish-DivTeam" # DivTeam doesn't bring languages. Usually Original + Spanish.
```

Comment:
Remove `-DivTeam`, this shouldn't have been added in https://github.com/Jackett/Jackett/pull/9655/commits/25689907dd343c4fc2728f07104a07d555919f36